### PR TITLE
Implement send token within and across blockchain application - Closes #4402, #4401

### LINKED ISSUE #4402: Implement send token across application
### Description

- Update send native and LSK token across application component according to new 
     - Native token [design](https://www.figma.com/file/Krew1j8uTJVfb6RuojImTK/Implement-interoperable-token-usage?node-id=149%3A11347)
     - LSK token [design](https://www.figma.com/file/Krew1j8uTJVfb6RuojImTK/Implement-interoperable-token-usage?node-id=387%3A19217)
- Fetch and display active supported blockchain applications
     - Display only active applications (this information is similar to application management, except here we only show the names of the applications)
- Fetch and display supported tokens for a given application
     - Filter tokens which has `zero` balance
- Reuse recipient address input and adopt to new design
     - All existing validation logic should be retained
- Display the transaction fee token symbol based on application configuration
- Adopt transaction summary according to new design
- Include `Enter your password` screen in the send flow
- Adopt transaction result screen according to new design
- Reuse the `Add message` input and retain the existing logic
     - Update to new design
- Display `initialisation fee` when recipient address selected token balance is `zero`
     - The `initialisation fee` displayed should be corresponding native token 
- Display `ccm fee` in LSK
     - If the send account doesn’t have enough LSK, then display error message

### Acceptance Criteria

- Set LSK token as default in the token dropdown
- User should be able to send LSK tokens from Mainchain to Sidechain
- User should be able to send supported Native tokens selected from the dropdown
- User should be able to sign and broadcast transaction to network (Mock)
- User should be able to see the token balance
- User should receive feedback when failed to send transaction to network (Mock)

### Additional information

- The logo and token symbol can be hardcoded for now and updated once the service API is ready
- Mock RPC endpoint `token_getBalances` to return the following:
	```
{
  balances: {
    tokenID: string,
    availableBalance: string,
    lockedBalances: { 
       moduleID: number, 
       amount: string 
    }[],
  }[],
} []
```
Use application management Mock API to display supported applications

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #4402 and #4401.

### How was it solved?

- [x] Refactor the Send form to the new UI designs (i.e to support within and across chain token sending)
- [x] Implement enter password page for transaction signing.
- [x] Refactor the transaction summary page in the multi-step
- [x] Refactor the Transaction status page in the multi-step
- [x] Refactor the unit tests
- [x] Refactor e2e tests

`Note:` this branch was updated with [feature/4034-change-sdk-api](https://github.com/LiskHQ/lisk-desktop/tree/feature/4034-change-sdk-api) and as a result most of the changes aren't yet in feature/token-usage

### How was it tested?

e2e
jest

### CODE REVIEW

File: src/modules/account/components/ManageAccounts/ManageAccounts.js [hunk 1]
```diff
@@ -4,14 +4,15 @@ import { withRouter } from 'react-router';
 
 import Box from 'src/theme/box';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import Dialog from '@theme/dialog/dialog';
```
Comment (ikem-legend) [inline src/modules/account/components/ManageAccounts/ManageAccounts.js]: @eniolam1000752 This import can be updated in order to maintain consistency with similar imports from the same location

File: src/modules/dpos/validator/components/RegisterDelegateForm/index.js [hunk 2]
```diff
@@ -38,10 +38,14 @@ const RegisterDelegateForm = ({
   const network = useSelector(state => state.network);
   const [error, setError] = useState('');
   const [loading, setLoading] = useState(false);
-  const [username, setUsername] = useState(prevState?.rawTx?.asset.username ?? '');
+  const [username, setUsername] = useState(prevState?.rawTx?.params.username ?? '');
 
-  const onConfirm = (rawTx) => {
-    nextStep({ rawTx });
+  const onConfirm = (rawTx, trnxData, selectedPriority, fees) => {
```
Comment (ikem-legend) [inline src/modules/dpos/validator/components/RegisterDelegateForm/index.js:43]: @eniolam1000752 `trnxData` is never used but it's in the function definition

File: src/modules/dpos/validator/components/VoteSummary/voteSummary.test.js [hunk 3]
```diff
@@ -105,10 +111,9 @@ describe('VotingQueue.Summary', () => {
     const wrapper = mountWithRouter(Summary, props);
 
     expect(wrapper).toContainMatchingElement('VoteStats');
-    expect(wrapper).toContainMatchingElement('.fee-value');
+    expect(wrapper).toContainMatchingElement('.fee-value-Transaction');
```
Comment (ikem-legend) [inline src/modules/dpos/validator/components/VoteSummary/voteSummary.test.js]: @eniolam1000752 `Transaction` should be renamed to small case for consistency

File: src/modules/transaction/components/TransactionPriority/TransactionPriority.test.js [hunk 4]
```diff
@@ -65,7 +70,7 @@ describe('TransactionPriority', () => {
     wrapper.setProps({ ...props, token: tokenMap.LSK.key, selectedPriority: 3 });
     // simulate blur so that the edit icon is shown
     wrapper.find('.custom-fee-input').at(1).simulate('blur');
-    wrapper.find('span.fee-value').simulate('click');
+    wrapper.find('span.fee-value-Transaction').simulate('click');
```
Comment (ikem-legend) [inline src/modules/transaction/components/TransactionPriority/TransactionPriority.test.js]: @eniolam1000752 The full class should be in small letters

**Review by ikem-legend — DISMISSED** [review]

**Review by ikem-legend — APPROVED** [review]

**Review by soroushm — APPROVED** [review]

**Pull request merged**